docs/faqs.md

```diff
@@ -420,12 +420,20 @@ Please, read [Inactivity and user removal section](adding_packages/README.md#ina
 
 ## Can we add package which are parts of bigger projects like Boost?
 
-Sadly no. There have been many efforts in the past and we feel it's not sustainable given the number of combinations of libraries and version. See #14660 for recent discussions.
+Sadly no. There have been many efforts in the past and we feel it's not sustainable given the number of combinations of libraries and version.
+See #14660 for recent discussions. There is one main "boost" recipe with many versions maintained. Adding boost libraries with no dependencies
+just opens the does to graph resolution problems and once available allows for depedant libraries to be added.
 
-There is one main "boost" recipe with many versions maintained.
-
-There are good arguments for permitting some boost libraries but we feel doing so is not fair to the rest.
+In order to avoid this the sole permuation which permissible is when the project does not package any headers under the `boost/` folder, does not use the boost namespace
```

Comment:
```suggestion
In order to avoid this the sole permutation which is permissible is when the project does not package any headers under the `boost/` folder, does not use the boost namespace
```